On nixos x86_64, `moto` fails its tests:

```
           raise EndpointConnectionError(endpoint_url=request.url, error=e)
E           botocore.exceptions.EndpointConnectionError: Could not connect to the endpoint URL: "https://sts.amazonaws.com/"

/nix/store/x9g2ip5f3wbbrjd4b4qlqm6nngq2x7pc-python3.9-botocore-1.23.35/lib/python3.9/site-packages/botocore/httpsession.py:414: EndpointConnectionError
```

```
=========================== short test summary info ============================
FAILED tests/test_stepfunctions/test_stepfunctions.py::test_state_machine_list_executions_with_filter
====== 1 failed, 3114 passed, 1 skipped, 35 warnings in 70.91s (0:01:10) =======
```

--- a/pkgs/applications/networking/gns3/gui.nix
+++ b/pkgs/applications/networking/gns3/gui.nix
@@ -7,7 +7,21 @@ let
   ];
 
   python = python3.override {
-    packageOverrides = lib.foldr lib.composeExtensions (self: super: { }) defaultOverrides;
+    packageOverrides = lib.foldr lib.composeExtensions (self: super: {
+      jsonschema = super.jsonschema.overridePythonAttrs (oldAttrs: rec {
+        version = "3.2.0";
+
+        src = super.fetchPypi {
+          inherit (oldAttrs) pname;
+          inherit version;
+          sha256 = "sha256-yKhbKNN3zHc35G4tnytPRO48Dh3qxr9G3e/HGH0weXo=";
+        };
+
+        SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+        doCheck = false;
+      });
+    }) defaultOverrides;
   };
 in python.pkgs.buildPythonPackage rec {
   pname = "gns3-gui";

